types/Color.hpp

```diff
@@ -9,9 +9,10 @@
 
 namespace shared::types
 {
-    typedef Primary unsigned char;
-    typedef struct {
-        Color(Primary r, Primary g, Primary b, Primary a) : r(r), g(g), b(b), a(a){};
+    typedef unsigned char Primary;
+
+    typedef struct Color_s {
+        Color_s(Primary r, Primary g, Primary b, Primary a) : r(r), g(g), b(b), a(a) {}
```

Comment:
```suggestion
    typedef struct Color {
        Color(Primary r, Primary g, Primary b, Primary a) : r(r), g(g), b(b), a(a) {}
```

Reply:
warning: ‘typedef’ was ignored in this declaration
   14 |     typedef struct Color {